compiler+runtime/include/cpp/jank/runtime/core/math.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <jank/runtime/object.hpp>
+#include <jank/runtime/obj/big_decimal.hpp>
```

Comment:
We forward declare (on line 13) so that we don't need to include the header here. So let's remove this include.